es-app/src/views/gamelist/ISimpleGameListView.cpp

```diff
@@ -35,20 +36,7 @@ void ISimpleGameListView::onThemeChanged(const std::shared_ptr<ThemeData>& theme
 	mHeaderImage.applyTheme(theme, getName(), "logo", ALL);
 	mHeaderText.applyTheme(theme, getName(), "logoText", ALL);
 
-	// Remove old theme extras
```

Comment:
The `ThemeExtras` class was removed in the time since this grid view implementation was originally done.  This change needs to be reverted.